k8s/plugin/src/main.rs

```diff
@@ -15,17 +15,36 @@ struct CliArgs {
     operations: Operations,
 
     /// Kubernetes namespace of mayastor service
-    #[clap(global = true, long, short = 'n', default_value = "mayastor")]
-    namespace: String,
+    #[clap(global = true, long, short = 'n')]
+    namespace: Option<String>,
```

Comment:
Not sure about this, because this now modifies the help message and becomes unclear that mayastor is still the default namespace?

Reply:
Modified the docs now of -n flag.